site/public/js/grade-inquiry.js

```diff
@@ -223,11 +223,10 @@ function newDiscussionRender(discussion) {
     }
 }
 
-function previewInquiryMarkdown() {
+function previewInquiryMarkdown(mode) {
     const markdown_textarea = $(this).closest('.markdown-area').find('[name="replyTextArea"]');
     const preview_element = $('#inquiry_preview');
-    const preview_button = $(this);
     const inquiry_content = markdown_textarea.val();
 
-    previewMarkdown(markdown_textarea, preview_element, preview_button, { content: inquiry_content });
+    previewMarkdown(mode, markdown_textarea, preview_element, { content: inquiry_content });
```

Comment:
Instead of having a `previewMarkdown` function for the different places. Could we instead have a wrapper function that takes the `markdown_textarea` and `preview_element` variables and call `previewMarkdown` in that wrapper function instead so we won't have to edit all these functions across the system?